src/tiledb/cloud/vcf/ingestion.py

```diff
@@ -1004,6 +1046,18 @@ def ingest_manifest_dag(
             **kwargs,
         )
 
+    if metadata_uri:
+        sample_uris = submit(
+            read_metadata_uris_udf,
+            dataset_uri_result,
+            config=config,
+            metadata_uri=metadata_uri,
+            metadata_attr=metadata_attr,
+            verbose=verbose,
+            name="Read VCF URIs from metadata ",
+            **kwargs,
+        )
+
```

Comment:
seems like you want an `else` below this, otherwise you’ll be doing all this hard work just to immediately overwrite it

Reply:
It works as expected. I guess reusing the `sample_uris` variable is confusing. I can improve that.